org.flozz.yoga-image-optimizer.yml

```diff
@@ -0,0 +1,23 @@
+app-id: org.flozz.yoga-image-optimizer
+runtime: org.freedesktop.Platform
+runtime-version: "20.08"
+sdk: org.freedesktop.Sdk
+command: yoga-image-optimizer
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=host
```

Comment:
why `host`? it's very broad.

Reply:
It is maybe too much... What is the right permission to be able to read/write files from the user's home and removable media (lake flash drives, external hdd,...)?

Reply:
I started answering here but found that it would be best as a feature request https://github.com/flozz/yoga-image-optimizer/issues/6

I tested 1.0 and because of the way it works there is no way around this permission.

By default, it writes the output file next to the input file.

Reply:
Yes I answered on the issue you opened. The idea is interesting (and can probably became the default option), but I will not have time to work on this now, so it will have to wait a little :)

Maybe we can put a "WIP" on this PR until I have more time?

Reply:
I don't think this PR should be blocked by `--filesystem=host` but it's not up to me.